tests/libvmi/vmi.go

```diff
@@ -248,7 +248,7 @@ func WithCPUFeature(featureName, policy string) Option {
 }
 
 func WithPasstInterfaceWithPort() Option {
-	return WithInterface(InterfaceDeviceWithPasstBinding([]v1.Port{{Port: 1234, Protocol: "TCP"}}...))
+	return WithInterface(InterfaceWithPasstBindingPlugin([]v1.Port{{Port: 1234, Protocol: "TCP"}}...))
```

Comment:
This option is located in the wrong file (vmi.go), please take the opportunity to move it.

Reply:
done.